admin/src/graphql/searchUsers.js

```diff
@@ -1,14 +1,17 @@
 import gql from 'graphql-tag'
 
 export const searchUsers = gql`
-  query ($searchText: String!) {
-    searchUsers(searchText: $searchText) {
-      userId
-      firstName
-      lastName
-      email
-      creation
-      emailChecked
+  query ($searchText: String!, $currentPage: Int, $notActivated: Boolean) {
+    searchUsers(searchText: $searchText, currentPage: $currentPage, notActivated: $notActivated) {
+      userCount
+      userList {
+        userId
+        firstName
+        lastName
+        email
+        creation
+        emailChecked
+      }
```

Comment:
the page size should be a parameter and a globally defined constant for now - else we have to change it in trillions of places